ddd-koans/Value Object/Static Factory/task.html

```diff
@@ -0,0 +1,37 @@
+<html>
+    <p>
+        You may have noticed that in Self Validation task we throw exception -> this behaviour breaks the flow. <br/>
+        This is not the best thing to do so we would change the way we create TimeSignatures to be sure that exceptions are not thrown.
+    </p>
+
+    <p>
+        First of all, we are going to use a concept that origins in functional programming, but will
+        help us in here. So, if you don't know <i>sealed classes</i> in Kotlin, read the
+        <a href="https://kotlinlang.org/docs/reference/sealed-classes.html">documentation</a>.
+    </p>
+
+    <p>
+        As you can see in the code in the example we changed our TimeSignature class to ValidTimeSignatures and
+        extend the newly created sealed class TimeSignature. We have also created object InvalidTimeSignature that also
+        extends TimeSignature. Now you are going to implement <b>create</b> method that creates ValidTimeSignature or returns
+        InvalidTimeSignature object.
+    </p>
+
+    <p>
```

Comment:
I would suggest creating a separate task for `of` factory method.

Reply:
Done